ipamd/ipamd.go

```diff
@@ -153,13 +153,32 @@ func (c *IPAMContext) StartNodeIPPoolManager() {
 }
 
 func (c *IPAMContext) updateIPPoolIfRequired() {
+	c.reconcileENIIP()
```

Comment:
How often is `updateIPPoolIfRequired` called?  Is there the possibility of prolonging throttling congestion by reconciling to frequently? **CR** if so

Reply:
every 5 seconds

Reply:
That's pretty fast.  Imagine 500+ nodes each reconciling every 5 secs, if 10-20% are short, that's going to translate to 20 tps against EC2.  Do we know what default rate limit is?  I suspect 20 tps from one account is quite hard on the ec2 Network Manager, and this rate will congest over throttling.